fvtest/compilertriltest/ArithmeticTest.cpp

```diff
@@ -221,6 +222,48 @@ float fdiv(float l, float r) {
     return l/r;
 }
 
+bool isnan(float x) {
+    return x != x; // NANs never compare equal
+}
+
+bool isinf(float x) {
+    return std::abs(x) == std::numeric_limits<float>::infinity();
+}
+
+float copysign(float x, float y) {
```

Comment:
This seems to be the only problem on z/OS. `copysign` is already defined in `math.h` so we cannot redefine here. Either use the standard library function or rename this.